src/repositories/user.ts

```diff
@@ -0,0 +1,52 @@
+import { User } from '../models/User';
+import { CreateUser } from '../models/CreateUser';
+import { v4 as uuidv4 } from 'uuid';
+import { UpdateUser } from '../models/UpdateUser';
+import { ApplicationError } from '../models/application-error';
+
+const store = new Map<string, User>();
+
+export function getByIdFromDB(id: string): User {
+  const user: User | undefined = store.get(id);
+  if (user) {
+    return user;
+  }
+
+  throw new Error(`user ${id} not found`);
+}
+
+export function createUserInDB(newUser: CreateUser): string {
+  const id = uuidv4();
+  const user: User = {
+    ...newUser,
+    id,
+    isDeleted: false,
+  };
+
+  const userList = Array.from(store.values()).forEach((u: User) => {
```

Comment:
You do not use `userList` in the following code, so what is the purpose of this code?